src/main/scala/pikeman/guesser/strategy/BinaryGuesserStrategy.scala

```diff
@@ -0,0 +1,26 @@
+package pikeman.guesser.strategy
+
+import pikeman.GameNumbers
+import pikeman.guesser._
+
+class BinaryGuesserStrategy extends GuesserStrategy {
+
+  private var min = GameNumbers.Min
```

Comment:
Not a serious issue at all but you have a small duplication when you make the initialization of the class and when the round changes. You might find interesting to have a representation of the range of possible options as you add more code to this repo.